zanata-model/src/main/java/org/zanata/model/HAccount.java

```diff
@@ -108,7 +102,7 @@ public HPerson getPerson() {
     }
 
     @NaturalId
-    @UserPrincipal
+//    @UserPrincipal
```

Comment:
Probably best to note that alongside the commenting. But that's just nitpicking.